packages/extension/cypress/e2e/trustlines.cy.ts

```diff
@@ -158,7 +158,7 @@ describe('Trustline', () => {
 
     // Login
     cy.get('input[name="password"]').type(PASSWORD);
-    cy.contains('button', 'Unlock').click();
+    cy.contains('button', 'Unlock').click({ force: true });
```

Comment:
@FlorianBouron it works locally without this param, but not in the CI...
In the CI, I get the error
```
     CypressError: Timed out retrying after 4050ms: `cy.click()` failed because this element:

`<button class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-jcwuwq-MuiButtonBase-root-MuiButton-root" tabindex="0" type="button">...</button>`

is being covered by another element:

`<iframe style="position: fixed; top: 0px; left: 0px; width: 100%; height: 100%; border: none; z-index: 2147483647;"></iframe>`

Fix this problem, or use {force: true} to disable error checking.
```